ddp-workspace/projects/ddp-brain/app.yaml

```diff
@@ -0,0 +1,13 @@
+service: brain
+runtime: python37
+
+handlers:
+  - url: /(.*\.(css|eot|gz|html|ico|js|map|png|svg|ttf|woff|woff2|jpg|jpeg|gif|png|json|pdf))
+    static_files: dist/\1
+    upload: dist/(.*\.(css|eot|gz|html|ico|js|map|png|svg|ttf|woff|woff2|jpg|jpeg|gif|png|json|pdf))
+    secure: always
+
+  - url: /(.*)
+    static_files: dist/index.html
+    upload: dist/index.html
+    secure: always
```

Comment:
All these app.yml files look the same except for the `service` name.
Using `python37` as all examples I saw used `python27`, but docs prominently say they will be dropping support for Python 2 at some point.
(We don't use Python or Java or node or anything, but this is necessary. And depending of which one you runtime you select, required/optional/etc configuration contents of files vary).

Reply:
I think we can template-ize this with ctmpl.  Look at `docker-compose-study.yaml.ctmpl` and `config/manifest-study.rb`  to get a sense of how we isolate some of this copy-pasta with ctmpl.

Reply:
https://golang.org/pkg/text/template is the rabbit hole to go down